deps/build.jl

```diff
@@ -1,8 +1,17 @@
-if Sys.KERNEL === :Darwin && (!success(`command -v xcode-select`) || isempty(readchomp(`xcode-select -p`)))
+using Compat
```

Comment:
needs to be in REQUIRE

Reply:
Ah, good catch. I just assumed we already had a Compat dependency.